installer/volcano-development.yaml

```diff
@@ -136,7 +136,6 @@ spec:
             - --tls-cert-file=/admission.local.config/certificates/tls.crt
             - --tls-private-key-file=/admission.local.config/certificates/tls.key
             - --ca-cert-file=/admission.local.config/certificates/ca.crt
-            - --admission-conf=/admission.local.config/configmap/volcano-admission.conf
```

Comment:
This configuration is useful and should not be deleted :)

Reply:
Why delete admission-conf?

Reply:
![image](https://user-images.githubusercontent.com/13084946/130430205-1a76ed8e-add9-4529-89c9-24b3e498fca3.png)

I encountered this error.

Reply:
same issue with https://github.com/volcano-sh/volcano/issues/1663, latest image should be updated.
/cc @Thor-wl

Reply:
> same issue with #1663, latest image should be updated.
> /cc @Thor-wl

Got it. It seems to be necessary to update recently.

Reply:
> Why delete admission-conf? BTW, please add self signature when push the commit.

Thanks for reminding, I added my signature.
I also encountered this error. so i  updated YAML, is right? :)

Reply:
Well, thanks for the contribution. But the root reason is not here. I'll fix it.

Reply:
thanks